SharedPackages/DesignResourcesKit/Sources/DesignResourcesKit/Colors/SharedPalettes/SharedDefaultColorPalette.swift

```diff
@@ -0,0 +1,164 @@
+//
+//  SharedDefaultColorPalette.swift
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+
+#if canImport(AppKit)
+
+/// Based on DefaultColorPalette, for compatibility. This class will be removed as soon as `FigmaColorPalette` is validated
+struct SharedDefaultColorPalette: SharedColorPaletteDefinition {
```

Comment:
This Palette should 100% be the same as `DefaultColorPalette`, with extra values.

Added for compatibility / safety, goal is to remove this structure as soon as the UX is validated